src/Pages/Shared/ConfigMapSecret/Secret.utils.tsx

```diff
@@ -420,10 +420,18 @@ const handleValidJson = (isESO: boolean, json, dispatch: (action: ConfigMapActio
             secretStoreRef: json.secretStoreRef,
             refreshInterval: json.refreshInterval,
             esoData: null,
+            dataFrom: null,
+            template: null,
         }
         if (Array.isArray(json?.esoData)) {
             payload.esoData = json.esoData
         }
+        if (Array.isArray(json?.dataFrom)) {
+            payload.dataFrom = json.dataFrom
+        }
+        if (typeof json?.template === 'object' && !Array.isArray(json?.template)) {
```

Comment:
We should remove the 2nd `?` check
```suggestion
        if (typeof json?.template === 'object' && !Array.isArray(json.template)) {
```